File: program/program/find-area-of-a-cuboid/find_area_of_a_cuboid.pl
#!/usr/bin/perl
use warnings;
use strict;
print "Enter the length : ";
my $length = <STDIN>;
print "Enter the width : ";
my $width = <STDIN>;
print "Enter the height : ";
my $height = <STDIN>;
my $cuboid = 2 * ($length * $width) + ($length * $height) + ($width * $height);
print "Area of Cuboid: ".$cuboid, "\n";

File: program/program/find-area-of-a-cuboid/FindAreaOfACuboid.rb
pi = Math::PI

print "Enter length : \n"
length = gets.chomp.to_i
print "Enter width : \n"
width = gets.chomp.to_i
print "Enter height : \n"
height = gets.chomp.to_i

area = 2 * (length * width + length * height + width * height)
print "area : #{area}"
